iOS Client/TabController/MBTabBarController.swift

```diff
@@ -71,15 +67,18 @@ class MBTabBarController: UITabBarController, StoreSubscriber {
     }
     
     func newState(state: MBAppState) {
-        
+        if let podImageName = state.podcastsState.selectedPodcast?.image {
+            playPauseView.image.image = UIImage(named: podImageName)
```

Comment:
It seems like we could check if the image is already set, then don't do any work.

Reply:
would also need to track the old state to make sure we overwrite incorrect images